### PR TITLE
Ptombez pr

### PR DESCRIPTION
**Pull request by tschanzt**
@p-tombez a bit many commits since i had newer v11 branch. but can you check on my new commits?

### CODE REVIEW

File: mail_inline_css/models/mail_template.py [hunk 1]
```diff
@@ -1,22 +1,26 @@
 # Copyright 2017 David BEAL @ Akretion
```
Comment (simahawk) [inline mail_inline_css/models/mail_template.py]: add our copy too

File: mail_inline_css/demo/demo_template.xml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                </tr>
+              </tbody>
+            </table>
+          </div>
+          <main id="main_content">
+            <div class="greeting">
+              <p>Hello <span t-field="object.name" />.</p>
+            </div>
+            <div id="content">
+              <p>This e-mail styles are inline rendered although its template define styles as embedded CSS !</p>
```
Comment (asaunier) [inline mail_inline_css/demo/demo_template.xml]: define > defines
"CSS !" > "CSS!"

**Comment by tschanzt** [comment]
@asaunier @simahawk updated.

File: mail_inline_css/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Mail Inline CSS',
+    "summary": "Convert style tags in inline style in your mails",
+    'version': '11.0.1.0.0',
+    'author': 'Akretion, Odoo Community Association (OCA)',
```
Comment (simahawk) [inline mail_inline_css/__manifest__.py]: add Camptocamp

File: mail_inline_css/tests/test_mail_inline_styles.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Camptocamp SA
```
Comment (simahawk) [inline mail_inline_css/tests/test_mail_inline_styles.py]: 2019

**Comment by asaunier** [comment]
@tschanzt Thanks! OK from my perspective!

**Pull request merged**